packages/ember-metal/lib/meta.js

```diff
@@ -345,6 +346,14 @@ export class Meta {
       obj[key] = value;
     }
   }
+
+  setFactory(factory) {
```

Comment:
Any preference to function accessors vs normal accessors? I can be fine with either, was just surprised this isn't a accessor and am curious if there was a reason for it